tsconfig.json

```diff
@@ -21,5 +21,6 @@
     "jest.config.js",
     ".eslintrc.js",
     "typedoc.js"
-  ]
+  ],
+  "exclude": ["node_modules"]
```

Comment:
```sh
- info Linting and checking validity of types ..We detected TypeScript in your project and reconfigured your tsconfig.json file for you. Strict-mode is set to false by default.

The following suggested values were added to your tsconfig.json. These values can be changed to fit your project's needs:

        - exclude was set to ['node_modules']

```

Reply:
Excluding node_modules is not necessary.

When `exclude` is used together with `include` it will exclude things in the included directories/files. Since there are no node_modules in `src` or `tests` or `scripts` directories - there's no need to add it.